Source/WebCore/page/scrolling/ScrollingTree.cpp

```diff
@@ -56,6 +57,14 @@ struct CommitTreeState {
     HashSet<ScrollingNodeID> synchronousScrollingNodes;
     // orphanNodes keeps child nodes alive while we rebuild child lists.
     OrphanScrollingNodeMap orphanNodes;
+    // Hosted subtrees needing attaching to scrolling tree after main commit has finished
+    Vector<std::pair<LayerHostingContextIdentifier, Vector<std::unique_ptr<ScrollingStateTree>>>> pendingSubtreesNeedingCommit;
+    // Nodes that are descendants of a frame hosting node.
+    HashSet<ScrollingNodeID> hostedScrollingNodes;
+    // This has a value when doing a commit for a hosted subtree
+    RefPtr<ScrollingTreeFrameHostingNode> hostingNode;
+    // Identifier for the parent frame associated with this commit
+    FrameIdentifier frameId;
```

Comment:
"parent frame" gives me pause. Is it the parent frame, or this frame? How does it related to hostingNode?

Reply:
Right this is just the frame associated with the commit not the parent.